test/base.py

```diff
@@ -53,10 +62,6 @@ def setUpClass(self):  # noqa
         from app import app
         self.app = app
 
-    @classmethod
```

Comment:
why is this gone and does it need to be implemented somewhere (it wasn't previously)?